components/dashboard-modules/EnergyRecommendation.tsx

```diff
@@ -0,0 +1,74 @@
+import Image from 'next/image';
+import { FC } from 'react';
+import { useSiteData } from '~/lib/hooks';
+import { getCurrentTimeForecast } from '~/lib/utils';
+import content from '../../content/power-card-content.json';
+import NumberDisplay from './NumberDisplay';
+
+/**
+ * Determines the appliance with the greatest energy required that is less than or equal to the current output
+ * @param currentOutput the current output in kW based on the getCurrentTimeForecast
+ * @returns the index of the appliance with the greatest enrgy requiement less than or equal to the current ouput
+ * or -1 if an index could not be found
+ */
+
+const getBestRecommendationIndex = (currentOutput: number | undefined) => {
+  if (currentOutput != undefined) {
+    let maxIndex = null;
+    let maxKW = 0;
+
+    content.appliances.forEach((appliance, i) => {
+      if (
+        Number(appliance.kW) > maxKW &&
+        Number(appliance.kW) <= currentOutput
+      ) {
+        maxIndex = i;
+        maxKW = Number(appliance.kW);
+      }
+    });
+
+    return maxIndex;
+  }
+  return null;
+};
+
+const EnergyRecommendation: FC<{ siteUUID: string }> = ({ siteUUID }) => {
+  const { forecastData, isLoading } = useSiteData(siteUUID);
+  const currentOutput = forecastData
+    ? getCurrentTimeForecast(forecastData.forecast_values)
+    : undefined;
+
+  const recommendationIdx = currentOutput
+    ? getBestRecommendationIndex(currentOutput)
+    : null;
+
+  if (recommendationIdx === null) {
+    return <NumberDisplay title="Recommendations" value="N/A" />;
+  } else {
+    const appliance = content.appliances[recommendationIdx];
+    return (
+      <div
+        className="
+          flex-1
+          flex
+          my-2
+          p-4
+          text-center
+          bg-ocf-gray-1000
+          rounded-2xl"
```

Comment:
could these styles be changed a bit to keep the content within the module centered as screen size increases? 

<img width="453" alt="Screen Shot 2023-03-13 at 9 09 00 PM" src="https://user-images.githubusercontent.com/62641231/224866410-68787d73-4fb3-48da-a7ed-72ee77f660ee.png">